README.md

```diff
@@ -242,6 +242,8 @@ Send unhandled errors and performance metrics to sentry.
 
 Configure Sentry DSN using `[p]set api sentry dsn,https://fooo@bar.baz/9`, then load the Cog `[p]load sentry`.
 
+Requires `pip install -U sentry_sdk`.
```

Comment:
We shouldn't need to specify this in the documentation. The pip requirements are defined in the `info.json` file for the cog, and should be installed automagically when you run `[p]cog install <cog_name>`
Feel free to correct me if there's an issue with `sentry_sdk` not installing when the cog is installed.

Reply:
Did not know that was a feature. Dope.
I think it's just typo'd in [sentry/info.json#L13](https://github.com/rHomelab/LabBot-Cogs/blob/develop/sentry/info.json#L13) then. I'll kill this and fix that.